### ISSUE TITLE
Column offset bug in parenthesised Attribute nodes

### ISSUE THREAD

**Issue by lysnikolaou** [gvanrossum/pegen#202]
When parsing something like `(a).b`, pegen produces (part):

```
Attribute(
    value=Name(id="a", ctx=Load(), lineno=1, col_offset=1, end_lineno=1, end_col_offset=2),
    attr="b",
    ctx=Load(),
    lineno=1,
    *col_offset=0,*
    end_lineno=1,
    end_col_offset=5,
)
```

while the stdlib `ast` module produces:

```
Attribute(
    value=Name(id="a", ctx=Load(), lineno=1, col_offset=1, end_lineno=1, end_col_offset=2),
    attr="b",
    ctx=Load(),
    lineno=1,
    *col_offset=1,*
    end_lineno=1,
    end_col_offset=5,
)
```

As you can see, pegen points to the opening parenthesis, while CPython points to the first NAME of the Attribute. My guess is that this is a CPython bug, since the parentheses are both part of the Attribute, but is that correct?

**Comment by lysnikolaou** [comment]
Note that there is already an open issue for this on bpo: https://bugs.python.org/issue39474.

**Comment by gvanrossum** [comment]
The example in the bpo issue makes it clear why this is a real problem.

**Comment by lysnikolaou** [comment]
Fixed by python/cpython#18477.
